### DISCUSSION TITLE
Hello anh em nhé

### DISCUSSION THREAD

**Discussion by chungngx** [codingreshapefuture/Community#3] [General]
<h1>👋 Hế lô! </h1>
Đây là nơi để mọi người đưa ra ý tưởng về những dự án để thực hiện.
Mọi người cứ thoải mái để cử, cả ý tưởng, lẫn công nghệ, và cùng nhau trao đổi, lựa chọn nhá.

**Comment by lucthienphong1120** [comment]
hello
